### PR TITLE
Added more combinations to All-in-One

### PR DESCRIPTION
**Pull request by UZziell**
new combinations:
* Trojan-H2-TLS
* Vless-H2-TLS
* VMESS-H2-TLS
* ShadowSocks-TCP-TLS
* ShadowSocks-WS-TLS
* ShadowSocks-gRPC
* ShadowSocks-H2-TLS

Example links were added to README.
Example client configurations were also added.

### CODE REVIEW

File: All-in-One-fallbacks-Nginx/README.md [hunk 1]
```diff
@@ -1,41 +1,75 @@
...
-* Multiple domains could be used at the same time, including domains behind cloudflare CDN. (For cloudflare, make sure websocket and gRPC are enabled in Network section). In this configuration these domains are **example.com** and **behindcdn.com**
\ No newline at end of file
+* Multiple domains could be used at the same time, including domains behind cloudflare CDN. (For cloudflare, make sure websocket and gRPC are enabled in Network section). In this configuration these domains are **example.com** and **behindcdn.com**
+* HTTP2 inbounds (Trojan-H2, Vless-H2, VMESS-H2 and ShadowSocks-H2)
+    * [HTTP2 Transport does not support fallback based on `path`](https://xtls.github.io/config/transports/h2.html#http-2). That's why SNI is used instead.
+    * It's possible to create a CNAME dns record for all the H2 SNIs and use that as the address of the client config without setting custom SNI on client but it's optinal.
+
+    * It is assumed that the **example.com** domain has a **wildcard certificate**. If it's not a wildcard certificate or it it's a self-signed certificate, **allowInsecure** must be **ture** on client configuration.
```
Comment (yuhan6665) [inline All-in-One-fallbacks-Nginx/README.md:57]: typo in "it it's" "ture"

File: All-in-One-fallbacks-Nginx/server.json [hunk 2]
```diff
@@ -4,26 +4,53 @@
...
+      "tag": "Vless-TCP-XTLS",
       "port": 443, // This is TLS entrypoint. This entrypoint does the SSL Termination then routes the request based on the Path or ALPN type.
-      "protocol": "trojan",
+      "protocol": "vless",
       "settings": {
         "clients": [
           {
-            "password": "desdemona99",
+            "id": "90e4903e-66a4-45f7-abda-fd5d5ed7f797",
             "flow": "xtls-rprx-direct" //To enable XTLS Direct, this configuration must be added; otherwise, delete it. V2Ray has deleted the XTLS application since version v4.33.0. If you want to use this application, it is recommended to choose Xray.
```
Comment (yuhan6665) [inline All-in-One-fallbacks-Nginx/server.json:14]: we would recommend new configs to use vision instead of direct for the flow

Comment (UZziell) [reply All-in-One-fallbacks-Nginx/server.json]: Good to know. Switched to Vision Flow.

**Comment by yuhan6665** [comment]
Thanks so much for all your effort! LGTM

**Comment by UZziell** [comment]
This was the least I could do.
Thanks for the awesome project.🤟🏻🤟🏻

**Pull request merged**